lib/katello/plugin.rb

```diff
@@ -171,4 +171,11 @@
                    :partial => 'smart_proxies/show/storage',
                    :onlyif => proc { |proxy| proxy.has_feature?(SmartProxy::PULP_FEATURE) || proxy.has_feature?(SmartProxy::PULP_NODE_FEATURE) }
   end
+
+  RemoteExecutionFeature.register(:katello_package_install, N_("Katello: Install Package"), :description => N_("Install package via Katello interface"), :provided_inputs => ['package'])
```

Comment:
The feature represents a something that some part of code provides. It's declaring, that some part of the system is going to use remote execution for something. I've used the katello prefix, becuase it's really a Katello owning the feature.